Tests/SWBBuildSystemTests/ClangExplicitModulesTests.swift

```diff
@@ -373,7 +373,7 @@ fileprivate struct ClangExplicitModulesTests: CoreBasedTests {
         }
     }
 
-    @Test(.requireSDKs(.macOS))
+    @Test(.requireSDKs(.macOS), .disabled("setting global environment interferes concurrent tests"))
```

Comment:
Please add a reference to https://github.com/swiftlang/swift-build/issues/835 to track the permanent fix for this issue and re-enabling the test.